### ISSUE TITLE
CLI  points not showing 

### ISSUE THREAD

**Issue by oxatik** [nexus-xyz/nexus-cli#142]
![Screenshot (17)](https://github.com/user-attachments/assets/7173c17b-40ff-44ea-a57f-328d6f164f56)

**Comment by irfan-files** [comment]
yeah same as me vfx55782@gmail.com "EU2LMXKtA1hEmWdS8tfIUjoK4722"

**Comment by romdoni9** [comment]
![Screenshot_2024-12-11-08-51-54-101_com android chrome](https://github.com/user-attachments/assets/08649cc3-a563-4650-b86f-c812cc6a0bd7)
why p
![IMG_20241211_085234](https://github.com/user-attachments/assets/16d9c776-2284-4b76-8fd4-3dc4b760e2a5)
why my points are gone

**Comment by romdoni9** [comment]
![Screenshot_2024-12-11-02-25-39-165_com android chrome](https://github.com/user-attachments/assets/b2482d4e-5737-4bc8-96f1-235b2d260028)
k
![Screenshot_2024-12-11-02-25-46-779_com android chrome](https://github.com/user-attachments/assets/49186f05-1ab2-4894-bd14-404bb1e0f694)
hitwhy my points are gone

**Comment by SofiyaStark** [comment]
same 

![Screenshot - 2024-12-11T102901 329](https://github.com/user-attachments/assets/a9dea339-b316-4fb9-9548-6dba80b7faad)
